build.gradle

```diff
@@ -12,31 +12,40 @@ repositories {
 
 dependencies {
     testCompile group: 'junit', name: 'junit', version: '4.12'
-}
+    implementation 'com.google.code.gson:gson:2.8.5'
+    implementation 'org.asynchttpclient:async-http-client:2.8.1'
```

Comment:
We don't need this library anymore. You can remove it

Reply:
Okay